README.md

```
@@ -2,313 +2,1521 @@
+- [Ayanabha Misra](https://ayanabha.life)
+- [Aycan Öğüt](https://aycan.dev)
+- [Ayfri](https://ayfri.com)
+- [Ayoub Alahyane](https://aalahyan3.tech)
+- [Ayush Das](https://ayushdas.vercel.app) [Full Stack Developer]
+- [Ayush Kumar Gupta](https://ayyush08.vercel.app) [Full Stack Developer]
+- [Ayush Kumar](https://ayushj.is-a.dev)
+- [Ayush Nighoskar](https://ayushn.netlify.app)
+- [Azaan Suhail](https://personal-portfolio-website-seven-teal.vercel.app)
+- [Azel](https:omargpax.vercel.app) [Software Developer - Full Stack]
+- [Dev Mir](https://www.mrwajahatalimir.com) [App Developer | Python Programmer | Android Enthusiast]
+- [Ujjwal Shukla](https://ujjwalportfolio2.vercel.app)[Full Stack Developer | Ui/Ux Designer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://ujjwalportfolio2.vercel.app)